reg_tests/test_files/ablNeutralEdgeAMS/ablNeutralEdgeAMS.yaml

```diff
@@ -0,0 +1,332 @@
+            specific_dissipation_rate: 1.0
+
+        - alpha_upw:
+            velocity: 1.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - alpha:
+            velocity: 0.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - upw_factor:
+            velocity: 1.0
+            turbulent_ke: 0.0
+            specific_dissipation_rate: 0.0
+
+        - relaxation_factor:
+            velocity: 0.7
+            pressure: 0.3
+            turbulent_ke: 0.7
+            specific_dissipation_rate: 0.7
+
+        - noc_correction:
+            pressure: yes
+
+        - limiter:
+            pressure: no
+            velocity: yes
+            turbulent_ke: yes
+            specific_dissipation_rate: yes
+
+        - projected_nodal_gradient:
+            velocity: element
+            pressure: element
+            turbulent_ke: element
+            specific_dissipation_rate: element
+            ndtw: element
+
+        - turbulence_model_constants:
+            kappa: 0.40
+            tkeProdLimitRatio: 10.0 # 2015 Masson paper's c1 (I think based on comparing Nalu-Wind implementation eqns. and 2011 thesis)
+            betaStar: 0.03
+            cMu: 0.03
+            aOne: 0.31
+            betaOne: 0.0236
+            betaTwo: 0.0276
+            gammaOne: 0.3255
+            gammaTwo: 0.3011
+            sigmaKOne: 0.85
+            sigmaKTwo: 1.0
+            sigmaWOne: 0.5
+            sigmaWTwo: 0.67
+            SDRWallFactor: 0.625 # for timestepping not turbulence model
+            forcingFactor: 8.0 # for AMS
+            avgTimeScaleCoeff: 1.0 # for AMS
+
+        - source_terms:
+            momentum:
+              - EarthCoriolis 
+              - abl_forcing
+
+        - user_constants:
+            east_vector: [1.0, 0.0, 0.0]
+            north_vector: [0.0, 1.0, 0.0]
+            latitude: 51.343479
+            earth_angular_velocity: 7.2921159e-5
+            reference_velocity: 17.5
+            reference_height: 2500.0
+            roughness_height: 0.3
+            rans_below_ks: true
+            length_scale_limiter: yes
+
+        - input_variables_from_file:
+            velocity: velocity
+
+    output:
+      output_data_base_name: ablNeutralEdgeAMS.e
+      output_frequency: 10
+      output_node_set: no
+      output_variables:
+       - velocity
+       - average_velocity
+       - pressure
+       - pressure_force
+       - tau_wall
+       - turbulent_ke
+       - specific_dissipation_rate
+       - minimum_distance_to_wall
+       - turbulent_viscosity
+       - k_ratio
+       - rans_time_scale
+       - average_mass_flow_rate
+       - average_tke_resolved
+       - avg_res_adequacy_parameter
+       - resolution_adequacy_parameter
+       - metric_tensor
+       - element_courant
+       - average_production
+       - average_dudx
+       - forcing_components
+       - sst_f_one_blending
+       - l_t
+       - gamma
```

Comment:
let's remove `l_t` and `gamma`

Reply:
Removed them here and in `ablNeutralEdgeSST`.